hack/release.sh

```diff
@@ -25,7 +25,7 @@ function build_release() {
   local YAML_LIST="$(mktemp)"
   export TAG
   $(dirname $0)/generate-yamls.sh "${REPO_ROOT_DIR}" "${YAML_LIST}"
-  ARTIFACTS_TO_PUBLISH=$(cat "${YAML_LIST}" | tr '\n' ' ')
+  ARTIFACTS_TO_PUBLISH=$(uniq "${YAML_LIST}" | tr '\n' ' ')
```

Comment:
This will hide the real issue; generate-yamls.sh should be fixed instead.

Reply:
Also, uniq won't remove repeated items that are not in a row (which seems to be the case, judging by the output of the failed release job).